java/ql/test/ext/TopJdkApis/TopJdkApisTest.ql

```diff
@@ -0,0 +1,17 @@
+import java
+import TopJdkApis
+
+from string apiName, string message
+where
+  // top jdk api names for which there is no callable
+  topJdkApiName(apiName) and
+  not exists(Callable c | hasApiName(c, apiName)) and
```

Comment:
Small simplification:

```suggestion
  not hasApiName(_, apiName) and
```

Reply:
Thanks! Updated all in https://github.com/github/codeql/pull/11572/commits/a8ee633acd19861f94834effb24de72991f06414.